Spec/Auth.swift

```diff
@@ -1717,14 +1718,12 @@ class Auth : QuickSpec {
                     waitUntil(timeout: testTimeout) { done in
                         auth.authorise(nil, options: authOptions) { tokenDetails, error in
                             expect(authCallbackHasBeenInvoked).to(beTrue())
-
-                            authCallbackHasBeenInvoked = false
```

Comment:
> Test that defaults are changed when set (need initial + subsequent request without any arguments)

I guess you mean  `+ subsequent request`**`with`**`any arguments` here?

> Remove force completely

Remove it only from tests target, or also from the lib target?